common/locales/en/settings.json

```diff
@@ -74,7 +74,7 @@
     "usernameSuccess": "Login Name successfully changed",
     "emailSuccess": "Email successfully changed",
     "detachFacebook": "De-register Facebook",
-    "detachedFacebook": "Successully removed Facebook from your account",
+    "detachedFacebook": "Successfully removed Facebook from your account",
     "addedLocalAuth": "Successully added local authentication",
```

Comment:
I know it wasn't in the original issue, but we should fix it while we're doing this.